encrypted-media-respec.html

```diff
@@ -3029,7 +3029,7 @@ <h4>Media Data May Contain Encrypted Blocks</h4>
         <section id="initdata-encountered">
           <h4>Initialization Data Encountered</h4>
           <p>
-            The Initialization Data Encountered algorithm queues an <a def-id="encrypted"></a> event for <a def-id="initialization-data"></a> encounterd in the <a def-id="media-data"></a>.
+            The <dfn class="export">Initialization Data Encountered</dfn> algorithm queues an <a def-id="encrypted"></a> event for <a def-id="initialization-data"></a> encounterd in the <a def-id="media-data"></a>.
```

Comment:
Same comment here (and below for "Encrypted Block Encountered"): it seems better to me to have the dfn on the `<h4>` directly:

```html
<h4><dfn class="export">Initialization Data Encountered</dfn></h4>
```

Reply:
Done